src/script/Config.ts

```diff
@@ -43,7 +43,8 @@ export class Configuration {
   readonly ENVIRONMENT = env.ENVIRONMENT || 'production';
   readonly FEATURE = {
     ...env.FEATURE,
-    ENABLE_EXTRA_CLIENT_ENTROPY: env.FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY && Runtime.isWindows(),
+    ENABLE_EXTRA_CLIENT_ENTROPY:
+      env.FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY && (Runtime.isWindows() || env.FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY_FORCE),
```

Comment:
Can we rename this? Enable and force together make less sense to me. 
```suggestion
      env.FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY && (Runtime.isWindows() || env.FEATURE.FORCE_EXTRA_CLIENT_ENTROPY),
```